src/localization/de/learn.json

```diff
@@ -0,0 +1,163 @@
+  "steps.negativeLookahead.description": "Zum Beispiel möchten wir andere Zahlen als den Stundenwert im Text auswählen. Daher müssen wir den negativen Look-Ahead-Ausdruck `(?!)` nach unserem Ausdruck schreiben, um nur auszuwählen die numerischen Werte, die kein `PM` nach sich haben. Fügen Sie `PM` nach dem `!`-Zeichen in Klammern ein.",
+  
+  "steps.positiveLookbehind.title": "Positiver Lookbehind: `(?<=)`",
+  "steps.positiveLookbehind.description": "Zum Beispiel möchten wir den Preiswert im Text auswählen. Daher müssen wir, um nur die Zahlenwerte auszuwählen, denen `$` vorangestellt ist, den positiven Lookbehind-Ausdruck `(?< schreiben =)` vor unserem Ausdruck. Fügen Sie `\\$` nach dem `=`-Zeichen in den Klammern hinzu.",
+
+  "steps.negativeLookbehind.title": "Negativer Lookbehind: `(?&lt;!)`",
+  "steps.negativeLookbehind.description": "Zum Beispiel möchten wir im Text andere Zahlen als den Preiswert auswählen. Um daher nur numerische Werte auszuwählen, denen kein `$` vorangestellt ist, müssen wir das negative Lookbehind ` schreiben (?&lt;!)` vor unserem Ausdruck. Fügen Sie `\\$` nach dem `!` innerhalb der Klammern hinzu.",
+
+  "steps.flags.title": "Flaggen",
+  "steps.flags.description": "Flags ändern die Ausgabe des Ausdrucks. Aus diesem Grund werden Flags auch als `Modifikatoren` bezeichnet. Legt fest, ob der eingegebene Ausdruck Text als separate Zeilen behandelt, Groß-/Kleinschreibung beachtet oder alle Übereinstimmungen findet. Fahren Sie mit fort nächster Schritt, um die Flaggen zu lernen.",
+
+  "steps.flagsGlobal.title": "Globale Flagge",
+  "steps.flagsGlobal.description": "Das `global`-Flag bewirkt, dass der Ausdruck alle Übereinstimmungen auswählt. Wenn es nicht verwendet wird, wird nur die erste Übereinstimmung ausgewählt. Aktivieren Sie jetzt das `global`-Flag, um alle Übereinstimmungen auswählen zu können.",
+
+  "steps.flagsMultiline.title": "Mehrzeiliges Flag",
+  "steps.flagsMultiline.description": "Regex sieht den gesamten Text als eine Zeile. Aber wir verwenden das `multiline`-Flag, um jede Zeile separat zu behandeln. Auf diese Weise schreiben wir die Ausdrücke entsprechend dem Zeilenende für jede Zeile separat . Aktivieren Sie nun das `multiline`-Flag, um alle Übereinstimmungen zu finden.",
+
+  "steps.flagsCaseInsensitive.title": "Groß-/Kleinschreibung beachten",
+  "steps.flagsCaseInsensitive.description": "Um die Groß-/Kleinschreibung des von uns geschriebenen Ausdrucks zu entfernen, müssen wir das Flag `Groß-/Kleinschreibung ignorieren` aktivieren.",
+  
+  "steps.greedyMatching.title": "Gieriger Abgleich",
+  "steps.greedyMatching.description": "Regex führt standardmäßig einen Greedy-Match durch. Das bedeutet, dass das Matchmaking so lang wie möglich sein wird. Sehen Sie sich das Beispiel unten an. Es bezieht sich auf alle Matches, die auf „r“ enden und beliebig sein können vorangestelltes Zeichen. Aber es hört nicht beim ersten Buchstaben `r` auf.",
```

Comment:
I'm not sure if it's a correct word.

`Matchmaking`

Reply:
`Gieriger` == `Greedy` 
`Heiratsvermittler` == `Matchmaking`

Reply:
Abgleich is kinda like a common saying word!